### PR TITLE
add managing conflict guide

### PR DESCRIPTION
**Pull request by kne42**

### CODE REVIEW

File: content/maintainers/managing-conflict.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md) 
+as a starting point, tailoring it to your 
+community's specific needs. Larger projects should 
+have an impartial committee dedicated to handling 
+Code of Conduct violations, and a method for 
+community members to privately raise issues to 
+them.
+
+### Assume positive intent and acknowledge 
+negative impact
```
Comment (munkm) [inline content/maintainers/managing-conflict.md:30]: I think this needs to be moved up to the line above or you need to add the markdown header format for this to continue.

File: content/maintainers/managing-conflict.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Remind everyone that they share the same goals and 
+try to find commonalities between each party's 
+reasonings. Validate that perhaps the offending 
+party meant no malice while also validating the 
+ways that the offended party was hurt.
+
+### When to step in
+Once discussions begin to get heated, with 
+misunderstandings and accusations getting thrown 
+it, it is best to step in, ask people to cool off,
```
Comment (munkm) [inline content/maintainers/managing-conflict.md:44]:
```suggestion
in, immediate resolution can be difficult. At this point, 
project leaders should consider stepping in, asking 
involved parties to cool off, 
```

File: content/maintainers/managing-conflict.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+other.
+
+When the Code of Conduct is violated, you should 
+take disciplinary action as detailed therein and 
+communicate to the offending party what they did 
+wrong and why. Assure the offended party that you 
+are taking the offense seriously and detail what 
+actions you took to remedy the situation, as well 
+as taking into account what their desired outcome 
+is.
```
Comment (munkm) [inline content/maintainers/managing-conflict.md:55]: For CoC violations do we want to be explicit about protecting the identity of the reporter also?

File: content/maintainers/managing-conflict.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Do not wait for a report to take action on a Code 
+of Conduct violation, it's best to address obvious 
+breaches of the Code in a timely manner to 
+demonstrate that it is important and a priority of 
+your project. Staying proactive helps to cultivate
```
Comment (munkm) [inline content/maintainers/managing-conflict.md:61]: I love that you added this! Emphasizing timeliness is definitely something we want.

**Review by munkm — APPROVED** [review]
I really like this! 

For managing conflict do we want to recommend that projects include decision-making details in their governance also? Sometimes conflict arises based on technical decision making and ultimately the project needs to make a decision to move forward. Many projects have a consensus-seeking model, but explicitly having a path forward can help individuals move forwards too. 

For the CoC violations I think we're covering important basics. Do we also want to include information on community notifications (that is, that the project documents the violation and the resolution path?), anonymous reporting guidelines, and other practices?

File: content/maintainers/managing-conflict.md (hunk 3 above)
Comment (kne42) [reply content/maintainers/managing-conflict.md:55]: Yes!

**Pull request merged**